packages/core/tracing/package.json

```diff
@@ -67,6 +68,7 @@
   },
   "peerDependencies": {
     "@kong/kongponents": "9.30.0",
+    "@kong/sdk-konnect-js-internal": "^0.6.0",
```

Comment:
question: why is this a peerDependency?

Reply:
It looks like you're just utilizing type imports from the package? If that's the case this only needs to be included in devDependencies as it will be compiled away at build time

Reply:
Yepp, just using it for types. Will remove it from `peerDependencies` when I open the PR on shared-ui. Will be closing this PR then.